Description: Drop `-static` from LDFLAGS from tools/*/Makefile.am in order to
             avoid sleuthkit tools to be statically linked against libtsk.
Author: Cristian Greco <cristian@regolo.cc>
Reviewed-by: Joao Eriberto Mota Filho
Last-Update: 2013-12-04
--- a/tools/autotools/Makefile.am
+++ b/tools/autotools/Makefile.am
@@ -1,6 +1,5 @@
 AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall
 LDADD = ../../tsk/libtsk.la
-LDFLAGS += -static
 EXTRA_DIST = .indent.pro
 
 bin_PROGRAMS = tsk_recover tsk_loaddb tsk_comparedir tsk_gettimes
--- a/tools/fstools/Makefile.am
+++ b/tools/fstools/Makefile.am
@@ -1,6 +1,5 @@
 AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall 
 LDADD = ../../tsk/libtsk.la
-LDFLAGS += -static
 EXTRA_DIST = .indent.pro fscheck.cpp
 
 bin_PROGRAMS = blkcalc blkcat blkls blkstat ffind fls fcat fsstat icat ifind ils \
--- a/tools/hashtools/Makefile.am
+++ b/tools/hashtools/Makefile.am
@@ -1,6 +1,5 @@
 AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall 
 LDADD = ../../tsk/libtsk.la
-LDFLAGS += -static
 EXTRA_DIST = .indent.pro md5.c sha1.c
 
 bin_PROGRAMS = hfind
--- a/tools/imgtools/Makefile.am
+++ b/tools/imgtools/Makefile.am
@@ -1,6 +1,5 @@
 AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall
 LDADD = ../../tsk/libtsk.la
-LDFLAGS += -static
 EXTRA_DIST = .indent.pro
 
 bin_PROGRAMS = img_cat img_stat
--- a/tools/srchtools/Makefile.am
+++ b/tools/srchtools/Makefile.am
@@ -6,7 +6,6 @@
 
 sigfind_SOURCES = sigfind.cpp 
 sigfind_LDADD = ../../tsk/libtsk.la
-sigfind_LDFLAGS = -static
 
 indent:
 	indent *.c *.cpp
--- a/tools/vstools/Makefile.am
+++ b/tools/vstools/Makefile.am
@@ -1,6 +1,5 @@
 AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall 
 LDADD = ../../tsk/libtsk.la
-LDFLAGS += -static
 EXTRA_DIST = .indent.pro
 
 bin_PROGRAMS = mmls mmstat mmcat
